[WIP] Valmiiksi migratoitu harjadb

Jarzka opened the pull request.

Jarzka committed:
Luo pohja valmiiksi migratoidun imagen käyttämiselle

Jarzka committed:
Korjaa typo imagen nimessä

Jarzka committed:
Paranna kommenttia

Jarzka committed:
Varmista, ettei kontti kuole heti alkuunsa

Jarzka committed:
Korjaa typo

Kauko commented on test_envs/harjadev/Dockerfile:
```diff
@@ -12,7 +12,7 @@ RUN apt-get -y install maven
 
 # Konffataan pg kuuntelemaan verkkoa
-RUN sed -i "s/#listen_addresses.*/listen_addresses = '*'/g" /etc/postgresql/9.5/main/postgresql.conf
+RUN sed -i "s/#listen_addresses.*/listen_addresses = '*'/g" /e9tc/postgresql/9.5/main/postgresql.conf
```
Typo? /e9tc/ vs /etc/

Jarzka replied:
/etc/ kuuluu olla

Kauko commented on tietokanta/build_migrated_db_image.sh:
```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+docker build -t harjadb .
```
Mitä tämä skripti siis tekee?

Kauko commented:
Semmonen tuli kans mieleen, että jos meillä ei oo erityistä tarvetta käyttää vanhoja skriptejä, niin pitäiskö vaan nimetä nämä uudet dev_dbup.sh jne, ja uudelleennimetä vanhat esim dev_dbup_old.sh?

Kauko committed:
Buildaa image jos harjadb image puuttuu

Kauko committed:
Nimeä uudelleen vanha restart skrpti

Kauko committed:
Nimeä uudelleen uusi restart skirpti

Kauko committed:
Nimeä uudelleen vanha käynnistys skripti

Kauko committed:
Nimeä uudelleen uusi käynnistys skripti

Kauko committed:
Poista kommentit skriptistä

Kauko committed:
Päivitä tiedostopolut skripteihin

Kauko committed:
Komea newline vielä skriptiin

Jarzka committed:
Korjaa typo

Kauko committed:
Tulosta imagen tiedot myös ennen käynnistämistä

Kauko committed:
Käytä echossa oikeaa termiä, kontti tuhotaan, ei image

Kauko committed:
Älä turhaan luo testikantoja Dockerfilessa

Testikannat luodaan käyttämällä harja-kantaa templatena,
eli kannat pitää kuitenkin luoda uudelleen kun pääkantaan
on ajettu migraatiot + testidata.

Kauko committed:
Älä luo testidatan luomisen yhteydessä uudelleen olemassaolevia rooleja

Kauko committed:
Hio lokkausta

Kauko committed:
old_Devdb_up.sh käyttää myös vanhaa migraatio + testidata skriptiä

Kauko committed:
Poista vanhat docker skriptit

Kauko committed:
Merge branch 'develop' into valmiiksi_migratoitu_harjadb

Kauko committed:
Poista turhaksi jäänyt build_migrated_db_image.sh

Kauko committed:
Devdb_up ja _restart skripteille voi antaa oman imagen nimen

Image yritetään aluksi löytää lokaalisti, sitten docker hubista,
ja lopulta se buildataan itse.

Kauko committed:
Kokeile travis buildia dockerissa

Jarzka committed:
Peru Travis-muutos
